lib/src/mock-server/proxy.ts

```diff
@@ -35,8 +35,13 @@ export function proxyRequest({
     response.writeHead(res.statusCode ?? response.statusCode, res.headers);
     res.pipe(response);
   });
+  proxyRequest.on("error", e => {
+    console.error(`Failed to proxy request: ${e}`, e.stack);
+    response.statusCode = 500;
+    response.send();
+  });
 
-  if (incomingRequest.body) {
+  if (incomingRequest.body && Buffer.isBuffer(incomingRequest.body)) {
```

Comment:
After the change below with `express.raw` as per the documentation for this middleware, if there is no request body, `request.body` will be populated with `{}`, otherwise with a `Buffer` object. Why, no idea. Regardless, we only want to proxy the body through if it's actually a `Buffer` object.